ci-operator/config/openshift/os/openshift-os-master__okd-scos.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  ocp_builder_stream9:
+    name: builder
+    namespace: ocp
+    tag: stream9
+build_root:
+  image_stream_tag:
+    name: coreos-assembler
+    namespace: coreos
+    tag: latest
+images:
+- build_args:
+  - name: OPENSHIFT_CI
+    value: "1"
+  dockerfile_path: Containerfile
+  to: stream-coreos
+- build_args:
+  - name: VARIANT
+    value: okd-c9s
+  dockerfile_path: extensions/Dockerfile
+  inputs:
+    ocp_builder_stream9:
+      as:
+      - registry.access.redhat.com/ubi9/ubi:latest
+    stream-coreos:
+      as:
+      - registry.ci.openshift.org/rhcos-devel/rhel-coreos:latest
```

Comment:
this will build the okd node image on top of rhel-coreos, which is unwanted. 
We should use the c9s base image but we don't publish it to registry.ci as of now. 
We can change that though

Reply:
@jbtrystram this is now ready to go. Do you have any other concerns before approval?

Reply:
> this will build the okd node image on top of rhel-coreos, which is unwanted.

The default in then `Containerfile` of the os repo is a c9s base image (quay.io/openshift-release-dev/ocp-v4.0-art-dev:c9s-coreos) - and that is used to build the stream-coreos image which is then used to build the extensions. hope this makes it clear.

Reply:
We should probably use the same default in both TBH, but no need to block on that.